Shared/UI/Views/MessageContentView/MessageContentView.swift

```diff
@@ -69,8 +70,8 @@ class MessageContentView: BaseView {
     lazy var emotionCollectionView = EmotionCircleCollectionView(cellDiameter: self.cellDiameter)
     
     let emotionLabel = ThemeLabel(font: .regular)
-    let addEmotionButton = ThemeButton()
-    let addEmotionImageView = UIImageView(image: UIImage(systemName: "plus"))
+//    let addEmotionButton = ThemeButton()
```

Comment:
Hiding this for now as there is no API available for adding an attachment once a message has been sent. lame.